.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -1,18 +1,32 @@
-# This file contains a workflow which will be run on schedule every day, and
+# This file contains a workflow which will be run on schedule every day, 
+# or can be call from other workflows and
 # will collect all the eligible (PRs which are labelled "Ready to merge" by our 
-# script) python upgrade PRs from both edx and openedx orgs.
+# script) python upgrade PRs from openedx or edx org.
```

Comment:
Can you reword this to elaborate that this workflow when triggered from this repo will merge only openedx PRs and will merge edx orgs' PRs only when it will be called from the `edx/.github` repo?

Reply:
sure